inc/saibridge.h

```diff
@@ -326,6 +326,16 @@ typedef enum _sai_bridge_attr_t
     SAI_BRIDGE_ATTR_LEARN_DISABLE,
 
     /**
+     * @brief To disable flooding traffic (Broadcast, unknown unicast,
+     * unknown multicast) on a bridge
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_BRIDGE_ATTR_FLOOD_DISABLE,
```

Comment:
they are unrelated, learning mode is about the src mac, flood disable is about dst mac.